subsys/usb/Kconfig

```diff
@@ -79,6 +79,12 @@ config USB_DEVICE_REMOTE_WAKEUP
 	help
 	  This option requires USBD peripheral driver to also support remote wakeup.
 
+config USB_DEVICE_DRIVER_HANDLE_CTRL_EPIN_ZLP
+	bool
+	help
+	  Stack should not handle ZLP for Variable-length Data Stage
+	  bacause it is taken over by the hardware.
```

Comment:
Not by hardware but by LL drivers I think (for nRF for sure).

Reply:
There is comment in the nRF driver:  "a flag is set to prevent triggering status stage which is handled by hardware". If **Not by hardware but by LL drivers I think (for nRF for sure)** is true then please fix your driver so we do not have to do something ugly in the stack!